src/component/packaging/Packaging.jsx

```diff
@@ -1,16 +1,26 @@
 import React, { useEffect, useState } from "react";
 import axiosInstance from "../../axiosConfig";
+import { AgGridReact } from "ag-grid-react";
 import { Container } from "react-bootstrap";
 
+
 const Packaging = () => {
   const [packagingOrders, setPackagingOrders] = useState([]);
+  const [columnDefs] = useState([
+    { headerName: "Invoice No", field: "invoiceNo", sortable: true, filter: true },
+    { headerName: "Customer Name", field: "customerName", sortable: true, filter: true },
+    { headerName: "Product Name", field: "productName", sortable: true, filter: true },
+    { headerName: "Product Category", field: "productCategory", sortable: true, filter: true },
+    { headerName: "Quantity", field: "quantity", sortable: true, filter: true },
+    { headerName: "Status", field: "packagingStatus", sortable: true, filter: true },
+  ]);
+
   useEffect(() => {
     // Fetch packaging orders data from the backend
     const fetchPackagingOrders = async () => {
       try {
-        axiosInstance.get("/api/packaging").then(res => {
-            setPackagingOrders(res.data);
-        })
```

Comment:
both are same, please don't change in future again

Reply:
ok fine